webrender/src/renderer.rs

```diff
@@ -856,6 +861,11 @@ impl Renderer {
             // Restore frame - avoid borrow checker!
             self.current_frame = Some(frame);
         }
+        if let Some(ref mut notifier) = self.rendered_pipeline_notifier {
```

Comment:
Can we make the notifier callback just take &PipelineEpochMap? That way, the mem::replace() can be removed and the clear() below can be executed unconditionally, which seems tidier.